api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md

```diff
@@ -18,6 +18,8 @@ Each cell in the filter row contains a magnifying glass icon, pausing on which o
 
```

Comment:
What do you mean by "**pausing** on which opens a drop-down list"?
Does the drop-down list appear if you simply _hover over_ the magnifying glass icon?

Reply:
Yeah, hover is the right word.

Reply:
Okay, how about the following?

`Each cell in the filter row contains a magnifying glass icon. Hovering over this icon opens a drop-down list with filters [available for the column](/api-reference/_hidden/GridBaseColumn/filterOperations.md '{basewidgetpath}/Configuration/columns/#filterOperations').`